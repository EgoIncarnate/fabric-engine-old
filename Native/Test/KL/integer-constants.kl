function entry()
{
  report (Integer)2147483647;
  report (Integer)-2147483648;
}
