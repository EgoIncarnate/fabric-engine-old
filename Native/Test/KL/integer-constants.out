GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          CastNode( Integer )
            ConstSize( 2147483647 )
        Report
          CastNode( Integer )
            UniOp( - )
              ConstSize( 2147483648 )
2147483647
-2147483648
